db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_06_22_094606) do
+
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```

Comment:
- [ ] As a result of the missing block above :arrow_up: an error was equally flag on this line:
![Screenshot from 2023-06-23 23-33-29](https://github.com/Gmanlove/blog-app/assets/63087491/d52a3234-04a3-4e3e-9792-80847ad30153)